networkit/cpp/distance/BFS.cpp

```diff
@@ -20,12 +20,15 @@ void BFS::run() {
     reachedNodes = 1;
     sumDist = 0.;
 
+    constexpr auto infDist = std::numeric_limits<edgeweight>::max();
+    std::fill(distances.begin(), distances.end(), infDist);
+
     if (distances.size() < z) {
-        distances.resize(z, std::numeric_limits<edgeweight>::max());
+        distances.resize(z, infDist);
         visited.resize(z, ts);
     }
 
-    if (ts++ == 255) {
+    if (ts++ == std::numeric_limits<uint8_t>::max()) {
```

Comment:
This change also does nothing (I am indifferent about it).

Reply:
I think it's just better than having a magic number.